raiden/connection_manager.py

```diff
@@ -398,15 +398,14 @@ def _open_channels(self) -> bool:
             if channel_state not in funded_channels
         ]
         possible_new_partners = self._find_new_partners()
-        if possible_new_partners == 0:
```

Comment:
Removing this is the bugfix. As this check stopped the connection manager too early for a node that had a bunch of open channels detected but not enough of them funded to reach the target.